website/docs/cloud-docs/users-teams-organizations/organizations.mdx

```diff
@@ -212,6 +212,33 @@ Organization owners can set up an SSO provider for the organization.
 
 #### General
 
+##### Status Checks
+
+Status checks are notifications sent to your version control system VCS provider, providing details about specific commits, including the present status of the Terraform Cloud run. Please refer to your VCS provider's documentation regarding status checks (e.g., [GitHub Status Checks Documentation](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/collaborating-on-repositories-with-code-quality-features/about-status-checks)).
+
+Organization owners can choose between _aggregated_ (default) and _non-aggregated_ status checks. This setting determines whether detailed information and links are accessed directly from the VCS provider or Terraform Cloud.
+This setting also determines the number of status checks directly sent to the VCS Provider in response to actions such as pull or merge requests.
+
+###### Aggregated Status Checks
+
+Aggregated status checks offer a streamlined experience if you have a single repository containing configuration for many workspaces (a.k.a., a monorepo).
+
+When aggregated status checks are enabled, Terraform Cloud sends one VCS status check for all runs triggered by a VCS event. If multiple workspaces are tied to a shared repository, Terraform Cloud aggregates the status checks for these workspaces into one summary. This summary is unique to the workspace's organization and VCS client connection.
+
+You can access additional information about an aggregated status check in Terraform Cloud by clicking the **Details** link that the status check provides. This link navigates you to a Terraform Cloud page that offers the consolidated status check results across multiple workspaces, highlighting details such as resource changes and issues that require attention.
+
+![Screenshot: Organization Aggregated status checks](/img/docs/organization-vcs-general-aggregated-status-checks.png)
+
+###### Non-aggregated Status Checks
+
+The default behavior of VCS providers is that a provider receives a status for each triggered workspace and related run stage in response to a VCS event. For example, a VCS push triggers checks for each related workspace's run stages, including the plan operation, policy checks, cost estimation, run tasks, and more.
```

Comment:
Maybe change the beginning of the paragraph to `The alternative behavior` or something else?

Reply:
thanks for the catch. Went back and forth a few times on which is the "default"